internal/corazawaf/transaction.go

```diff
@@ -361,9 +362,18 @@ func (tx *Transaction) AddRequestHeader(key string, value string) {
 				tx.variables.requestCookies.AddCS(kl, k, v)
 			}
 		}
-	case "host":
-		// Retrieving from the host header the virtual host to populate serverName
-		tx.variables.serverName.Set(strings.Split(value, ":")[0])
+	case "host", ":authority":
```

Comment:
Reading the [RFC](https://www.rfc-editor.org/rfc/rfc9110.html#section-7.2-2), I think that also the `:authority` pseudo-header should be taken into account. Still reasoning about the best approach if both headers are present. Currently raising an error, but still setting the latest read value